cluster/gce/gci/configure-helper-common.sh

```diff
@@ -1467,79 +1467,28 @@ function start-collect-pprof {
     COMPONENTS_PORTS="8080"
     COMPONENTS_NAME="kube-apiserver"
     echo "Collecting kube-apiserver pprof at ${CURRENTTIME}"
-    collecting-pprof ${COMPONENTS_PORTS} ${COMPONENTS_NAME} 
+    ps -C kube-apiserver >/dev/null && collecting-pprof ${COMPONENTS_PORTS} ${COMPONENTS_NAME} || echo "kube-apiserver is not running on this machine"
```

Comment:
Somehow I feel line 1467, 1468, 1470 are overprogrammed :)

Reply:
how about directly use?
```
ps -C kube-apiserver >/dev/null && collecting-pprof "8080" "kube-apiserver" || echo "kube-apiserver is not running on this machine"
```

Reply:
> how about directly use?
> 
> ```
> ps -C kube-apiserver >/dev/null && collecting-pprof "8080" "kube-apiserver" || echo "kube-apiserver is not running on this machine"
> ```

I like this better :)